### ISSUE TITLE
docker: db not exists

### ISSUE THREAD

**Issue by jtagcat** [matrix-org/dendrite#1517]
```
postgres_1   | FATAL:  database "dendrite_roomserver" does not exist
postgres_1   | FATAL:  database "dendrite_account" does not exist
postgres_1   | FATAL:  database "dendrite_signingkeyserver" does not exist
postgres_1   | FATAL:  database "dendrite_federationsender" does not exist
postgres_1   | FATAL:  database "dendrite_account" does not exist
postgres_1   | FATAL:  database "dendrite_keyserver" does not exist
postgres_1   | FATAL:  database "dendrite_mediaapi" does not exist
postgres_1   | FATAL:  database "dendrite_syncapi" does not exist
postgres_1   | FATAL:  database "dendrite_appservice" does not exist
postgres_1   | FATAL:  database "dendrite_account" does not exist
```

https://github.com/matrix-org/dendrite/blob/master/build/docker/README.md prob needs some additions other than the database note.

**Comment by jtagcat** [comment]
Can modify https://github.com/matrix-org/dendrite/blob/master/docs/INSTALL.md a bit:

```
for i in account device mediaapi syncapi roomserver signingkeyserver federationsender appservice e2ekey naffka; do
    docker-compose -f docker-compose.deps.yml exec -u postgres postgres createdb -h postgres -U dendrite -O dendrite dendrite_$i
done
```
Note: postgres asks pw for each command.

**Comment by jtagcat** [comment]
Would probably be smart to also map `/var/lib/postgresql/data` somewhere...

**Comment by jtagcat** [comment]
Probably want ports added to the monolith docker conf

**Comment by jtagcat** [comment]
```
$ ~/ndock/dendrite$ curl https://localhost:8448 -k
404 page not found
$ ~/ndock/dendrite$ curl https://localhost:8008 -k
curl: (35) error:1408F10B:SSL routines:ssl3_get_record:wrong version number
```

stopping here, not going to do yet an another all-nighter for matrix

**Comment by jtagcat** [comment]
ref: https://github.com/matrix-org/dendrite/tree/master/build/docker
ref: https://github.com/matrix-org/dendrite/blob/master/build/docker/README.md

**Comment by jtagcat** [comment]
update while closing tabs, seems that dbs can be auto-created with:
```
    environment:
      POSTGRES_DB: dendrite_something
```

**Comment by jtagcat** [comment]
there should be mentions of well known as well

**Comment by jtagcat** [comment]
note to docs editor (me?): example conf is dendrite-conf.yaml, expected is dendrite.yaml

**Comment by neilalexander** [comment]
I have fixed a couple of things in commits 2f57853 and 73bc28b1 — the `create_db.sh` script was not executable in Git, therefore it wasn't getting run properly when starting PostgreSQL. I've also added a mention of the `/var/lib/postgresql/data` mount in the sample Compose config.

**Comment by jtagcat** [comment]
Thanks, the issue is now in my backlog, feel free to close / split off / whatever.
